CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,180 @@
+
+### Triagers
+
+Triagers help a project by reviewing issues and code for quality and correctness.
+They are knowledgeable about the project's codebase (in its entirety or a specific section)
+and software engineering principles.
+
+**Defined by:** "triage" permission in the project
+
+#### Pre-requisites
+
+- Community Member for at least 3 months
+- Helped to triage issues and pull requests
+- Knowledgeable about the codebase
+
+#### Responsibilities
+
+- Read through issues and PRs
+  - Answer questions when possible
+  - Add relevant labels
+  - Draw maintainers' attention (via `@mention`) if relevant
+  - Close issue (as "completed" or "not planned") if necessary
+- Help maintain project quality control via [code reviews] on PRs
+  - Focus on code quality and correctness, including testing and factoring
+  - May also review for more holistic issues, but not a requirement
+- Be responsive to review requests
+- May be assigned PRs to review if in area of expertise
+- Assigned test bugs related to the project of expertise
+
+#### Privileges
+
+- Same as for Community Members
+- Triager status may be a precondition to accepting large code contributions
+
+#### Promotion process
+
+- Sponsored by a maintainer
+  - With no objections from other maintainers
+  - Done through issue or PR to update the CODEOWNERS file
+- May self-nominate or be nominated by a maintainer
+  - In case of self-nomination, sponsor must comment approval on the issue/PR
+
+### Maintainers
+
+Maintainers are responsible for the project's overall health.
+They are the only ones who can approve and merge code contributions.
+While triage and code review is focused on code quality and correctness,
+approval is focused on holistic acceptance of a contribution including:
+
+- backwards/forwards compatibility
+- adherence to API and flag conventions
+- subtle performance and correctness issues
+- interactions with other parts of the system
+- consistency between code and documentation
+
+**Defined by:** "Maintain" permissions in the project and an entry in its CODEOWNERS file
+
+#### Pre-requisites
+
+- Triager for at least 3 months
+- Reviewed at least 10 substantial PRs to the codebase
+- Reviewed or got at least 30 PRs merged to the codebase
+
+#### Responsibilities
+
+- Demonstrate sound technical judgment
+- Maintain project quality control via code reviews
+  - Focus on holistic acceptance of contribution
+- Be responsive to review requests
+- Mentor contributors and triagers
+- Approve and merge code contributions as appropriate
+- Participate in OpenSSF or Scorecard-specific community meetings, if possible
+- Hosting Scorecard-specific community meetings, if possible and comfortable
+
+#### Privileges
+
+- Same as for Triager
+- Maintainer status may be a precondition to accepting especially large code contributions
+
+#### Promotion process
+- Sponsored by a maintainer
+  - With no objections from other maintainers
```

Comment:
Is there a certain time window maintainers should be given to respond? It would be a balance between giving maintainers a chance to respond (e.g. I was just on vacation for a week or two), but also not blocking if someone is too busy.

Reply:
Good point. I can add a minimum waiting period before the issue is closed, if you want. Two weeks?

Reply:
Probably best to leave as is, lest we over-specify. If it's a problem we can add it something later.